plugins/powerd/fu-plugin-powerd.c

```diff
@@ -31,6 +45,10 @@ fu_plugin_destroy (FuPlugin *plugin)
 gboolean
 fu_plugin_startup (FuPlugin *plugin, GError **error)
 {
+	if (!fu_plugin_powerd_delete_suspend_file(error)) {
+		g_prefix_error(error, "lock file unable to be deleted");
+		return FALSE;
+	}
 	FuPluginData *data = fu_plugin_get_data (plugin);
```

Comment:
Declarations must go before any other statements

Reply:
I was going to say why didn't something in CI catch this, but it did :)
```
../plugins/powerd/fu-plugin-powerd.c:52:2: error: ISO C90 forbids mixed declarations and code [-Werror=declaration-after-statement]
```